cache_path_test.go

```diff
@@ -12,6 +12,20 @@ import (
 	"github.com/bitrise-io/go-utils/pathutil"
 )
 
+func createDirStruct(t *testing.T, contentByPth map[string]string) {
+	for pth, content := range contentByPth {
+		dir := filepath.Dir(pth)
+		if err := os.MkdirAll(dir, 0777); err != nil {
```

Comment:
Are you sure this is the right file permission? With this, it will available for everybody.

Reply:
we usually use this permission, but i will change to:

0755 Commonly used on web servers. The owner can read, write, execute. Everyone else can read and execute but not modify the file.